regression/scripts/Vehicles/Concorde.py

```diff
@@ -360,6 +362,7 @@ def vehicle_setup():
     
     # setup
     turbojet.number_of_engines = 4.0
+    turbojet.wing_mounted = [True] * int(turbojet.number_of_engines)
```

Comment:
Since this is a default, we should find a way to avoid needed to modify it for the number of engines unless there's a case where only some engines are wing mounted.

Reply:
I could just set the default argument to True, meaning all engines are mounted on the wing, and if this is not the case then the user can input a list of booleans, indicating which engines are not. Do you think that's a good idea?

Reply:
FINC